schema/rdl_schema_0.1.json

```
@@ -1025,6 +788,104 @@
 				"Judgement",
 				"Hybrid"
 			]
+		},
+		"Spatial": {
+			"title": "Spatial coverage",
+			"type": "object",
+			"description": "Information about the spatial coverage of the dataset.",
+			"properties": {
+				"countries": {
+					"title": "Countries",
+					"type": "array",
+					"description": "The countries covered by the geographical area, from the open country codelist (ISO 3166-1).",
```

Not sure why I put 'open' in the proposal. Country should be a closed codelist. Once https://github.com/GFDRR/rdl-standard/pull/110 is merged and we merge `dev` into this branch and run `./manage.py pre-commit` the link I added will exist.

```suggestion
					"description": "The countries covered by the geographical area, from the closed [country codelist](https://rdl-standard.readthedocs.io/en/dev/data_model/codelists.html#country).",
```

Also, since the codelist is closed, need to add an `enum`.